### DISCUSSION TITLE
Idea, Help for Knob code needed

### DISCUSSION THREAD

**Discussion by OstertagM** [Bodmer/TFT_eSPI#975] [Q&A - General] [answered]
I want to program some knob like attached pictures:
Some experience I have to program but I don't how to work this your library.
I hope somebody can help me. It should also work with touch.

![grafik](https://user-images.githubusercontent.com/26908656/106510539-5ad74f80-64cf-11eb-914b-d8369d08dd74.png)
![grafik](https://user-images.githubusercontent.com/26908656/106511073-07b1cc80-64d0-11eb-996e-acd8a4b95ffc.png)

if I have based code when I can perhaps adapt it to make all like this:
[https://radmie.github.io/ng-knob/](url)

**Comment by Bodmer** [comment, marked as answer]
You could use the function in [this example sketch.](https://github.com/Bodmer/TFT_eSPI/tree/master/examples/320%20x%20240/TFT_ArcFill)

Another option is to [adapt the sketch here](https://github.com/Bodmer/TFT_eSPI/tree/master/examples/480%20x%20320/TFT_ring_meter).
